src/agoradatatools/great_expectations/gx/expectations/proteomics.json

```diff
@@ -21,7 +21,7 @@
       "expectation_type": "expect_column_values_to_match_regex",
       "kwargs": {
         "column": "uniqid",
-        "regex": "^^\\w{2,}|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2}$"
+        "regex": "^\\w{2,}|[A-NR-Z][0-9]([A-Z][A-Z0-9]{2}[0-9]){1,2}$"
```

Comment:
Is the extra "\" in "^\\w{2,}" a typo or an escape character? I can't tell

Reply:
That is an escape character! The extra `^` was a typo, though!